src/openms/source/ANALYSIS/ID/PeptideIndexing.cpp

```diff
@@ -368,9 +368,9 @@ PeptideIndexing::PeptideIndexing() :
 DefaultParamHandler("PeptideIndexing")
   {
 
-    defaults_.setValue("decoy_string", "_rev", "String that was appended (or prefixed - see 'prefix' flag below) to the accessions in the protein database to indicate decoy proteins.");
+    defaults_.setValue("decoy_string", "dec_", "String that was appended (or prefixed - see 'prefix' flag below) to the accessions in the protein database to indicate decoy proteins.");
 
-    defaults_.setValue("prefix", "false", "If set, protein accessions in the database contain 'decoy_string' as prefix.");
+    defaults_.setValue("prefix", "true", "If set, protein accessions in the database contain 'decoy_string' as prefix.");
```

Comment:
can we replace this flag with
`registerStringOption_("decoy_string_position", "<enum>", "prefix", "Should the 'decoy_string' be prepended (prefix) or appended (suffix) to the protein accession?", false);`
as is done in DecoyDatabase?

Would make things more consistent.
(I know its an 'API' change)